mPowerSDK/Startup/APHAppDelegate.m

```diff
@@ -339,13 +339,17 @@ - (NSMutableDictionary *)updateOptionsFor5OrOlder:(NSMutableDictionary *)initial
 
 - (NSArray *)allSetTextBlocks
 {
-    NSArray *allSetBlockOfText = nil;
+    NSString *tutorialTextInitial = @"This is your app to use as you choose. We will give you a quick tutorial to help you get started.";
```

Comment:
The tutorial is a nice idea but since it was not included in IRB submission, we cannot implement it at this time.